### DISCUSSION TITLE
what can I do to improve performance

### DISCUSSION THREAD

**Discussion by Masber** [akinomyoga/ble.sh#525] [Q&A]
Dear ble.sh community,

I just installed ble.sh and I am finding it an interesting project with a lot of value add to bash.
However, I find my terminal sluggish.
I currently use alacritty as a terminal and really see a difference between having ble.sh enabled or not.

My question is, is there anything I could do to improve performance?

thank you

**Comment by akinomyoga** [comment]
@Masber I wrote an article about it in `ble.sh`'s wiki page: [_Performance - akinomyoga/ble.sh Wiki_](https://github.com/akinomyoga/ble.sh/wiki/Performance).
